examples/partially_blindrsa.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2024 Cloudflare, Inc.
 // Licensed under the Apache-2.0 license found in the LICENSE file or at https://opensource.org/licenses/Apache-2.0
-import sjcl from '../src/sjcl/index.js';
+import sjcl from 'sjcl';
+import { Buffer } from 'node:buffer';
```

Comment:
avoid using `node:buffer` as it's not a standard API

Reply:
ok,